File: Benchmarks/enterprise/java/spring/testcode/BenchmarkTest47582.java
// SPDX-License-Identifier: Apache-2.0
import jakarta.servlet.http.HttpServletRequest;
import jakarta.servlet.http.HttpServletResponse;
import java.nio.file.Files;
import java.nio.file.Paths;
import org.springframework.web.bind.annotation.*;

@RestController
public class BenchmarkTest47582 {

    private static String expandTabs(String v) { return v.replace("\t", " "); }

    @GetMapping("/BenchmarkTest47582")
    public void BenchmarkTest47582(@RequestParam("id") String id, HttpServletRequest request, HttpServletResponse response) throws Exception {
        String userId = id != null ? id : "";
        String data = expandTabs(userId);
        if (System.currentTimeMillis() > 1000000000000L) {
            Files.write(Paths.get("/var/uploads/" + data), "data".getBytes());
        }
        response.setContentType("application/json");
        response.getWriter().print("{\"id\":0}");
    }
}

File: Benchmarks/enterprise/python/fastapi/testcode/benchmark_test_47582.py
# SPDX-License-Identifier: Apache-2.0
from fastapi import Request


async def BenchmarkTest47582(request: Request):
    auth_header = request.headers.get('authorization', '')
    data = bytearray(int(auth_header) if str(auth_header).isdigit() else 0)
    return {"updated": True}
